src/main/java/com/patternpedia/api/entities/user/UserEntity.java

```diff
@@ -20,29 +24,35 @@
 @Entity
 @Data
 @NoArgsConstructor
-@TypeDef(name = "pgsql_enum", typeClass = PostgreSQLEnumType.class)
+//@NamedQuery(name = "UserEntity.findAll", query="select u from UserEntity u order by u.role.name")
```

Comment:
is this line stil needed? (By the way, I'd prefer to collect all queries in the service that interacts with the repository - in the best case, you can use the repository functionalities thatare provided by Spring)